src/plugins/pbjx/components/revert-view-modal/RevertDetails.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import isPlainObject from 'lodash/isPlainObject'
+import RevertPropertiesTable from '../revert-properties-table';
+import filterData from '../../utils/filterData';
+import findNodeDiff from '../../utils/findNodeDiff';
+
+/**
+ * This will keep the whole object or list for a key. The reason is trying
+ * to revert specific keys in a block list or multiselect list is difficult
+ * because the data might have changed greatly. So we just put it back
+ * completely to the original state.
+ * 
+ * @param {Object} newNode 
+ * @param {Object} origNode 
+ */
+const fullMapsAndLists = (newNode, origNode) => {
```

Comment:
@rsumilang getting an eslint warning on this guy

Reply:
not getting it on mine